spec/soda_machine_spec.rb

```diff
@@ -18,6 +21,7 @@
     context "when the soda is in the machine" do
       it "returns the first instance of the soda brand the user requested" do
         expect(soda_machine.find_soda('Mountain Dew')).to eq(mountain_dew)
+        #pass in a string but equals an object
```

Comment:
We pass `find_soda` a string representing the brand of soda that we want and expect it to return the first `Soda` object with that brand.

That is, `find_soda` takes a string and returns a `Soda`.